src/Database/Behaviors/Purgeable.php

```diff
@@ -4,10 +4,14 @@ class Purgeable extends \Winter\Storm\Extension\ExtensionBase
 {
     /**
      * @var array List of attribute names which should not be saved to the database.
-     *
-     * public $purgeable = [];
      */
+    public $purgeable = [];
```

Comment:
It was being reported as a missing variable otherwise, but yeah I realise this is probably incorrect since purgeable needs to exist on the model. Will adjust.